### PR TITLE
Workload Component.GitHub.VisualStudio removed from VS 2022

### PR DESCRIPTION
**Pull request by Anawaz**
Component.GitHub.VisualStudio Workload removed from VS2022

### CODE REVIEW

**Comment by PRMerger16** [comment]
@Anawaz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: docs/install/create-an-offline-installation-of-visual-studio.md [hunk 1]
```diff
@@ -145,10 +145,17 @@ Open a command prompt and use the bootstrapper's parameters as defined in the [U
 > To prevent an error, make sure that your full installation path is less than 80 characters.
 
 - For .NET web and .NET desktop development, run:
-
+::: moniker range=">=vs-2019"
```
Comment (anandmeg) [inline docs/install/create-an-offline-installation-of-visual-studio.md:148]:
```suggestion
::: moniker range="<=vs-2019"
```

**Review by anandmeg — CHANGES_REQUESTED** [review]
@Anawaz, thank you for contributing to our docs.

**Comment by PRMerger18** [comment]
@Anawaz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by anandmeg — APPROVED** [review]
@Anawaz, thank you again for contributing to our docs.

**Pull request merged**